src/lib/expansion-panel/expansion-panel-adapter.ts

```diff
@@ -35,10 +63,25 @@ export class ExpansionPanelAdapter extends BaseAdapter<IExpansionPanelComponent>
     this._contentElement.addEventListener('transitionend', this._transitionEndListener);
   }
 
+  public addContentSlotListener(listener: EventListener): void {
+    this._defaultSlotElement.addEventListener('slotchange', listener);
+  }
+
   public addHeaderListener(type: keyof HTMLElementEventMap, listener: EventListener): void {
     this._headerElement.addEventListener(type, listener);
   }
 
+  public removeTriggerListeners(reset = false): void {
```

Comment:
Just a small code style request when it comes to parameters of methods. Using the "config object" pattern can typically lead to more readable code. Something like this:
```typescript
public removeTriggerListeners({ reset } = { reset: false }): void {
```

Then when calling it things are bit more clear:
```typescript
this._adapter.removeTriggerListeners({ reset: true });
```